src/connector/src/parser/protobuf/parser.rs

```diff
@@ -250,30 +250,38 @@ pub fn from_protobuf_value(field_desc: &FieldDescriptor, value: &Value) -> Resul
             ScalarImpl::Utf8(enum_symbol.name().into())
         }
         Value::Message(dyn_msg) => {
-            let mut rw_values = Vec::with_capacity(dyn_msg.descriptor().fields().len());
-            // fields is a btree map in descriptor
-            // so it's order is the same as datatype
-            for field_desc in dyn_msg.descriptor().fields() {
-                // missing field
-                if !dyn_msg.has_field(&field_desc)
-                    && field_desc.cardinality() == Cardinality::Required
-                {
-                    let err_msg = format!(
-                        "protobuf parse error.missing required field {:?}",
-                        field_desc
-                    );
-                    return Err(RwError::from(ProtocolError(err_msg)));
+            if let Some(DataType::Jsonb) = type_expected {
+                // The message is of type `Any`
```

Comment:
IIUC, if user defines a column as `JSONB`, you will always parse them as `Any`, right?

I am wondering there must be some place that recognizes the `google.protobuf.Any` type name and then output the correct JSON value. But I didn't see it in this PR. Is this inside the parsing library?

Reply:
+1 I have the same question.

Reply:
> IIUC, if user defines a column as JSONB, you will always parse them as Any, right?

Yes, in `protobuf_type_mapping`, if the incoming message's `MessageDescriptor` contains both `type_url` & `value`, the type will be interpreted as `Jsonb`, which will then be passed through the `from_protobuf_value`.